### ISSUE TITLE
tags$script() should automatically not escape content

### ISSUE THREAD

**Issue by wch** [rstudio/htmltools#75]
Currently, the content of `tags$script()` needs to be wrapped with `HTML()` in order to not be escaped. But there's no reason (that I can think of) that the content should be escaped, and it just causes confusion with users.

I realize that this is a departure from how all other tag functions work, but it's also true that the `<script>` tag is different from other tags.

**Comment by bborgesr** [comment]
I agree... But just to be clear, once we make this change, if the content is still wrapped in `HTML()`, there's no harm done, right? (important for backward compatibility and for people that don't realize that there's been a change...)

**Comment by timelyportfolio** [comment]
It would be great if all my old `tags$script(HTML(...))` work.  Thanks so much for all your work on this.

**Comment by jcheng5** [comment]
I didn't take this step in the past because I thought it best to be explicit about where strings were _not_ going to be escaped, and therefore, where you might be vulnerable to code injection attacks. However, on second thought it doesn't matter in this case--we're already talking about a script tag, so a malicious user wouldn't need to worry about `<` or `>` characters, they're literally already in the context they want to be to inject code.

So yeah, I guess there is not a reason to not do this.
